react/features/display-name/components/web/DisplayNameBadge.tsx

```diff
@@ -1,22 +1,16 @@
-// @flow
-
 import { makeStyles } from '@material-ui/core/styles';
-import React from 'react';
-
-type Props = {
+import * as React from 'react';
```

Comment:
why does the import need to be this way now?

Reply:
Added in tsconfig "allowSyntheticDefaultImports": true, that I think is going to fix this.